api/v1beta2/types.go

```diff
@@ -92,8 +92,5 @@ type Subnet struct {
 // VPCEndpoint describes a VPCEndpoint.
 type VPCEndpoint struct {
 	Address *string `json:"address"`
-	// +optional
-	FIPID *string `json:"floatingIPID,omitempty"`
```

Comment:
Not comfortable changing the spec, can we deprecate this field meanwhile and remove later in beta3 version or later..

Reply:
will do.

Reply:
Done.